spring-backend/src/main/kotlin/com/example/sztoswro/member/MemberService.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.sztoswro.member
+
+import com.example.sztoswro.exceptions.NoContentException
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder
+import org.springframework.stereotype.Service
+
+@Service
+class MemberService(val memberTmpRepo: MemberTmpRepo){//(val members: MemberRepository) {
+
+    fun getAll(): Iterable<MemberDAO> { return memberTmpRepo.getAll()} //members.findAll()
+
+    fun getMember(id: Long): MemberDAO {
+        return memberTmpRepo.findById(id) ?: throw NoContentException("Member with id = {id} not found.")
+    }
+
+    fun findMember(username: String): MemberDAO {
+        return memberTmpRepo.findByUsername(username) ?: throw NoContentException("Member with id = {id} not found.")
+    }
+
+    fun addMember(memberDAO: MemberDAO) {
+        val aUser = memberTmpRepo.findById(memberDAO.id)
```

Comment:
Dodawając usera nie znamy jego ID albo generujemy to w SERWISIE (to jest jednak jakaś logika biznesowa) albo zostawiamy to ORM'owi

Reply:
Tak jak pisałem w innym komentarzu: na razie szukamy po id, póki nie mamy połączenia z bazą. Potem będziemy szukać po username'ach.